### ISSUE TITLE
Does data augmentation change data label?

### ISSUE THREAD

**Issue by bilgan** [matterport/Mask_RCNN#903]
If i do random augmentation like padding, do i need to change data label (coordinates)?

`sometimes(iaa.CropAndPad(
            percent=(-0.05, 0.1),
            pad_mode=ia.ALL,
            pad_cval=(0, 255)
        )),`

`model.train(dataset_train, dataset_val,
                learning_rate=config.LEARNING_RATE,
                epochs=30,
                augmentation=augmentation,
                layers='heads')`

**Comment by umair-sabir** [comment]
this algorithm does it automatically

**Comment by bilgan** [comment]
Thanks Umair!

Another question, i have 300 labelled image. I flipped it different directions and calculated new labels for them. So now i have 900 labelled images. 

The question is should i need to use those already flipped images or do the augmentation (flipping, rotating or whatever) with original 300 images during the training?
